contrib/vms/README.md

```diff
@@ -28,9 +28,7 @@ distributions.
 For further details, consult the shell scripts `setup.sh` and
 `ci/install_dependencies.sh`.
 
-
-All boxes come with `conan` installed via pip in the `vagrant` user's home
```

Comment:
Why this change, did you touch the Vagrantfile?

Reply:
Because it was not true (At least I think so). We are not using conan in the boxes but instead we install the 3rd party libraries via the package manager of each system.

Reply:
That's only partially correct, the vagrant boxes come with all the dependencies installed via the distributions package manager, but they also come with conan pre-installed (check this script out: `contrib/vms/setup_user.sh`).

I have added conan into the mix, so that anyone starting such a box has a full development environment with all pieces ready to go.

Reply:
Ouch, I did not notice that. Sorry, I will revert this change.